src/components/molecules/OakJauntyAngleLabel/OakJauntyAngleLabel.tsx

```diff
@@ -4,11 +4,13 @@ import { OakBox, OakBoxProps } from "@/components/atoms";
 
 export type OakJauntyAngleLabelProps = {
   label: string;
+  id?: string;
 } & Omit<OakBoxProps, "onClick" | "label">;
 export const OakJauntyAngleLabel = (props: OakJauntyAngleLabelProps) => {
-  const { label, ...oakBoxProps } = props;
+  const { label, id, ...oakBoxProps } = props;
   return (
     <OakBox
+      id={id}
```

Comment:
Should this already work if `OakJauntyAngleLabelProps` has `Omit<OakBoxProps, "onClick" | "label">` which in turn has `id`?